packages/dialog/src/web/dialog-web.web.tsx

```
@@ -0,0 +1,48 @@
+    prop: openProp,
+    defaultProp: defaultOpen,
+    onChange: onOpenChangeProp,
+  });
+
+  return (
+    <RootContext.Provider value={{ open, onOpenChange }}>
+      <Dialog
+        defaultOpen={defaultOpen}
+        open={open}
+        onOpenChange={onOpenChange}
+        children={children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)